src/item/Trident.php

```diff
@@ -69,7 +69,6 @@ public function onReleaseUsing(Player $player, array &$returnedItems) : ItemUseR
 			$item = $entity->getItem();
 			$item->applyDamage(1);
```

Comment:
I feel like it would make more sense to damage the item before passing it to TridentEntity constructor instead of having to get/set it here